src/vs/workbench/parts/terminal/electron-browser/terminal.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as os from 'os';
+import platform = require('vs/base/common/platform');
+import processes = require('vs/base/node/processes');
+
+const powershellPath = `${ process.env.SystemRoot }/system32/WindowsPowerShell/v1.0/powershell.exe`;
+const isAtLeastWindows10 = platform.isWindows && parseFloat(os.release()) >= 10;
```

Comment:
I think it would be best to be explicit here, `parseFloat('10.2.22222')` may work but it feelss dodgy to me and the behavior itself may even be undefined and only works by coincidence.